podman_compose.py

```diff
@@ -928,7 +928,15 @@ def volume_ls(self, proj=None):
         volumes = output.splitlines()
         return volumes
 
-def normalize_service(service):
+def normalize_service(service, sub_dir=''):
+    if sub_dir and 'build' in service and 'context' in service['build']:
+        context = service['build']['context']
+        if re.match('^(((\./)?(\w+)+(/\w+)+?$)|(\.$))', context):
```

Comment:
I typically compile my regex once, not per-function call.

Reply:
I believe the regex is not needed at all, it should be just `.startswith('./')`

Reply:
you can hit the path to the context without a './'